src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -272,7 +272,7 @@ def update_address_ids(
     units_that_move_ids: pd.Index,
     starting_address_id: int,
     address_id_col_name: str,
-) -> Tuple[pd.DataFrame, pd.DataFrame, int]:
+) -> Tuple[pd.DataFrame, int]:
```

Comment:
Since you made this change, want to update the `Returns` section of the docstring below by deleting the "pop" line?

Reply:
Good call, I'll fix that in my next PR.